app/src/internal/java/com/kickstarter/viewmodels/FeatureFlagsViewModel.kt

```diff
@@ -1,23 +1,24 @@
 package com.kickstarter.viewmodels
 
 import androidx.annotation.NonNull
-import com.kickstarter.libs.ActivityViewModel
-import com.kickstarter.libs.CurrentConfigType
-import com.kickstarter.libs.CurrentUserType
-import com.kickstarter.libs.Environment
-import com.kickstarter.libs.ExperimentsClientType
+import com.kickstarter.libs.*
+import com.kickstarter.libs.utils.extensions.SEGMENT_ENABLED
+import com.kickstarter.model.FeatureFlagsModel
 import com.kickstarter.ui.activities.FeatureFlagsActivity
 import rx.Observable
 import rx.subjects.BehaviorSubject
 
 interface FeatureFlagsViewModel {
-    interface Inputs
+    interface Inputs {
+        fun updateSegmentFlag(flag: Boolean)
```

Comment:
Can this function be documented as well?

Reply:
Done